frontend/views/pages/Payments.vue

```diff
@@ -111,14 +118,23 @@ export default {
   },
   data () {
     return {
+      form: {
+        search: ''
```

Comment:
Just a small suggestion: The name "search" here suggested an action, and I think "searchText" was more clear about the intent.